lib/beaker/host/unix/exec.rb

```diff
@@ -85,23 +85,24 @@ def reboot(wait_time=10, max_connection_tries=9, uptime_retries=18)
       @logger.debug("Original Boot Time: #{original_boot_time}")
       @logger.debug("Current Boot Time: #{current_boot_time}")
 
+      # This actually may fail a few times before succeeding because virtual
+      # machines can skew backwards in time so we need to let the retry happen.
       unless current_boot_time > original_boot_time
         raise Beaker::Host::RebootFailure, "Boot time did not reset. Reboot appears to have failed."
       end
-    rescue Beaker::Host::RebootFailure => e
+    rescue ArgumentError => e
+      raise Beaker::Host::RebootFailure, "Unable to parse time: #{e.message}"
```

Comment:
Is this really fatal? Where does it even originate from?

Reply:
@ekohl This gets thrown if the regex match on the time doesn't work. If we get something that's absolute nonsense, we can't compare times.